pycbc/events/coinc.py

```diff
@@ -353,15 +353,15 @@ def cluster_coincs(stat, time1, time2, timeslide_id, slide, window, argmax=numpy
     cindex: numpy.ndarray
         The set of indices corresponding to the surviving coincidences.
     """
+
     logging.info('clustering coinc triggers over %ss window' % window)
 
     if len(time1) == 0 or len(time2) == 0:
         logging.info('No coinc triggers in one, or both, ifos.')
         return numpy.array([])
 
-    indices = []
     if numpy.isfinite(slide):
-        time = (time2 + (time1 + timeslide_id * slide)) / 2
+        time = (time1 + time2 + timeslide_id * slide) / 2
```

Comment:
add comment to explain why the time shift is added (and not subtracted) - eg 'for a time shifted coinc, time1 is greater than time2 by approximately timeslide_id * slide' / 'adding this quantity gives a mean coinc time located around time1'